astro_tests/dags/dbt_example_dags.py

```diff
@@ -89,7 +89,7 @@
 
             deps = DbtDepsOperator(
                 task_id=f"{project['project']}_install_deps",
-                project_dir=f"/usr/local/airflow/dbt/{project['project']}",
+                project_dir=f"/usr/local/airflow/dags/dbt/{project['project']}",
```

Comment:
It might make sense to make this a relative path instead of an absolute path. That would make this example work in more environments (especially when considering local development)

Reply:
@dimberman what if we just put the env var for the dags directory in the beginning of the path. Would that accomplish the same thing? Example:

```python

dags_folder = os.environ["AIRFLOW__CORE__DAGS_FOLDER"]

DbtDepsOperator(
  ...,
  project_dir=f"{dags_folder}/dbt/{project['project'}"
)
```